src/frontend/web_application/test/functional/protractor.conf.js

```diff
@@ -58,52 +57,54 @@ const cfg = {
     { package: 'protractor-console-plugin', failOnWarning: false, logWarnings: false, exclude: [/Warning:/] },
   ],
   logLevel: 'INFO',
-  debug: true,
+  debug: false,
 };
 
 if (process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY) {
   // platform config: https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/
-  const branch = (process.env.TRAVIS_PULL_REQUEST_BRANCH.length > 0) ?
-    process.env.TRAVIS_PULL_REQUEST_BRANCH :
-    process.env.TRAVIS_BRANCH;
 
   cfg.sauceUser = process.env.SAUCE_USERNAME;
   cfg.sauceKey = process.env.SAUCE_ACCESS_KEY;
+  cfg.sauceSeleniumAddress = process.env.SAUCE_ADDRESS + ':' + process.env.SAUCE_PORT + '/wd/hub';
+  cfg.sauceSeleniumUseHttp = true;
+  const branch = process.env.DRONE_BRANCH;
+
   cfg.multiCapabilities = [
     {
-      browserName: 'firefox',
```

Comment:
for saucelabs env : `firefox only` (saucelabs only provides firefox 45 for linux)